### PR TITLE
Create create-jira-ticket-on-issue.yml

### PR DESCRIPTION
**Pull request by fireridlle**

### CODE REVIEW

**Review by parkhomenko — DISMISSED** [review]

**Review by evbruno — DISMISSED** [review]

File: .github/workflows/create-jira-ticket-on-issue.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Set event vars
+        id: event
+        run: |
+          if [ ${{github.event.issue}} ]; then 
+            echo "::set-output name=SUMMARY::GITHUB ISSUE #${{github.event.issue.number}} ${{github.event.issue.title}}"
+            echo "::set-output name=DESC::${{github.event.issue.url}} \n\n ${{github.event.issue.body}}"
```
Comment (macnealefauna) [inline .github/workflows/create-jira-ticket-on-issue.yml:27]: github.event.issue.html_url

File: .github/workflows/create-jira-ticket-on-issue.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        uses: JamesSingleton/is-organization-member@1.0.0
+        id: isOrgMember
+        with:
+          username: ${{ steps.event.outputs.USER }}
+          organization: fauna
+          token: ${{ secrets.GH_TOKEN }}
+
+      - name: Create ticket
+        id: create
+        uses: atlassian/gajira-create@master
```
Comment (macnealefauna) [inline .github/workflows/create-jira-ticket-on-issue.yml:54]: s/master/main/

File: .github/workflows/create-jira-ticket-on-issue.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        with:
+          project: DRV
+          issuetype: ${{steps.event.outputs.ISSUE_TYPE}}
+          summary: ${{steps.event.outputs.SUMMARY}}
+          description: ${{steps.event.outputs.DESC}}
+          fields: '{"labels": ["github", "${{steps.event.outputs.LABEL}}"], "components": [{ "name": "C# Driver" }]}'
+
+      - name: Set triage state
+        if: ${{ steps.create.outputs.issue }}
+        uses: atlassian/gajira-transition@master
```
Comment (macnealefauna) [inline .github/workflows/create-jira-ticket-on-issue.yml:65]: s/master/main/

**Review by macnealefauna — DISMISSED** [review]

File: .github/workflows/create-jira-ticket-on-issue.yml (hunk 3 above)
Comment (cleve-fauna) [reply .github/workflows/create-jira-ticket-on-issue.yml:65]: actually needs to be master

File: .github/workflows/create-jira-ticket-on-issue.yml (hunk 1 above)
Comment (cleve-fauna) [inline .github/workflows/create-jira-ticket-on-issue.yml:27]:
```suggestion
            echo "::set-output name=DESC::${{github.event.issue.html_url}} \n\n ${{github.event.issue.body}}"
```

**Review by doom-weaver — APPROVED** [review]

**Pull request merged**